### PR TITLE
feat(server): add socket.io server

### PR DESCRIPTION
**Pull request by agory**
Add dependance sockect io, create a serverSocket to init and configure the server, connect the
socket.io server to the express/nextJs server

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,9 +25,15 @@
     "opn": "^5.2.0",
     "prop-types": "^15.6.1",
     "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "react-dom": "^16.2.0",
+    "socket.io": "^2.0.4"
   },
   "devDependencies": {
+    "@storybook/addon-actions": "^3.3.14",
```
Comment (mfrachet) [inline package.json]: This diffs are not on master ? 🤔

File: src/server/socketServer.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-console */
+const socketIo = require('socket.io');
+
+const socketServer = (server) => {
```
Comment (mfrachet) [inline src/server/socketServer.js]: Missing the tests for this part 😛

File: package.json (hunk 1 above)
Comment (agory) [reply package.json]: I added dependency for socket.io which moved some dependency to the top of the file.

**Comment by agory** [comment]
Hello @mfrachet, I have updated the pr. What do you think of the changes?

**Pull request closed without merging**